packages/icons/src/img/sprite.svg

```diff
@@ -2031,5 +2031,8 @@
         clip-rule="evenodd"
       />
     </symbol>
+    <symbol viewBox="0 0 64 64" id="lp-icon-osmo">
+      <path d="M32.62 60.61c-.445 0-.892-.254-1.147-.7-.191-.444-.191-.889.128-1.27l13.582-18.62-23.785 9.66c-.191.063-.319.126-.51.126-.574 0-1.084-.38-1.212-.89-.191-.508.064-1.08.51-1.397l20.916-12.266-36.794-2.097c-.765-.063-1.211-.635-1.211-1.27 0-.573.382-1.208 1.211-1.272l36.794-2.097-20.916-12.265c-.446-.318-.701-.89-.51-1.398.192-.509.638-.89 1.212-.89.191 0 .319.063.51.127l23.785 9.66L31.601 5.067c-.255-.381-.32-.89-.128-1.271.191-.445.638-.7 1.148-.7.319 0 .638.128.893.382L60.55 30.36c.383.381.574.953.574 1.462 0 .508-.191 1.016-.574 1.461L33.514 60.23a1.261 1.261 0 0 1-.893.38Z"/>
+    </symbol>
```

Comment:
```suggestion
    <symbol viewBox="0 0 20 20" id="lp-icon-osmo">
      <path d="M10.194 18.94a.42.42 0 0 1-.359-.218c-.06-.139-.06-.278.04-.397l4.245-5.819-7.433 3.019c-.06.02-.1.04-.16.04a.389.389 0 0 1-.378-.278c-.06-.16.02-.338.16-.438l6.535-3.832-11.498-.656a.394.394 0 0 1-.378-.397c0-.179.12-.377.378-.397l11.498-.655-6.536-3.833c-.14-.1-.219-.278-.16-.437.06-.16.2-.278.38-.278.06 0 .1.02.159.04l7.433 3.018-4.245-5.839a.408.408 0 0 1-.04-.397c.06-.139.2-.218.36-.218.099 0 .198.04.278.119l8.45 8.4c.119.12.179.298.179.457 0 .16-.06.318-.18.457l-8.449 8.42c-.08.08-.18.12-.279.12Z"/>
    </symbol>
```

Reply:
Awesome, thanks! I definitely agree on the consistency of the definition bit.